portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,12 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
+/**Servlet that return a greeting for web visitors*/
+@WebServlet("/greeting")
 public class DataServlet extends HttpServlet {
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println("Hi Abraham!");
```

Comment:
Is this supposed to be a message for users who visit your site? It may be good for this to be addressed to a wider audience. Maybe something like, "Welcome to the website of Abraham Haros!".

Reply:
Yes, actually that won't be static, this file was pretended to be use as the comment servlet, but I was testing how to handle multiple servlets. This file will be deleted.